app/views/generic_files/upload/_alerts.html.erb

```diff
@@ -1,17 +1,10 @@
-<div class="alert alert-info hide" id="success">
```

Comment:
It was used before we auto forwarded, so it is never shown.  As far as I could see it was cruft and I removed it.